platform_plugin_aspects/sinks/course_overview_sink.py

```diff
@@ -58,7 +86,22 @@ def serialize_item(self, item, many=False, initial=None):
         detached_xblock_types = get_detached_xblock_types()
 
         location_to_node = {}
-        items = modulestore.get_items(course_key)
+
+        # This call gets the entire course tree in order, because the
+        # get_items call does not guarantee ordering. It does not return
+        # detached blocks, so we gather them separately below.
+        course_block = modulestore.get_course(
+            course_key, revision="rev-opt-published-only"
```

Comment:
where is that revision key coming from?

Reply:
It comes from here, where it has been unchanged for 10 yrs: https://github.com/openedx/edx-platform/blob/749e18bcee396755f411f46d79b25f42628af7d1/xmodule/modulestore/__init__.py#L68

I'll make it a local constant and document it better if Dave says this is an ok direction, but I'd rather not import it from modulestore since that means a shim function for the import, mocking it in tests, etc, which doesn't seem worth it since it would only guard against the very unlikely change in the string value.